core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -21,7 +21,7 @@
          ,is_system_db/1
         ]).
 -export([get_account_realm/1, get_account_realm/2]).
--export([is_account_enabled/1]).
+-export([is_account_enabled/1, maybe_disable_account/1]).
```

Comment:
I feel like we should start moving these functions out of `wh_util` and into `kz_account`